Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu
+
+RUN apt-get update && apt-get install -yq nodejs npm
```

Comment:
this will install something that is newer ... is that not good enough ?
(I'd hope it won't be downgraded in newer versions of ubuntu)